tools/SpectrePch.hpp

```diff
@@ -27,4 +26,7 @@
 // Include Brigand related headers
 #include <Utilities/TMPL.hpp>
 
+// For Blaze error handling (see SetupBlaze.cmake)
+#include <Utilities/BlazeExceptions.hpp>
```

Comment:
I think we need to include this before we include blaze. I'm on mobile so maybe I just can't see the file quite right